src/setRafTimeout.js

```diff
@@ -10,7 +10,7 @@ export default function setRafTimeout(callback, timeout = 0) {
       callback(now);
       currTime = -1;
     } else {
-      requestAnimationFrame(shouldUpdate);
+      if (typeof requestAnimationFrame !== 'undefined') requestAnimationFrame(shouldUpdate);
```

Comment:
doing `typeof requestAnimationFrame !== 'undefined'` for the conditional because using `!requestAnimationFrame` as the conditional still caused the undefined error. This ensures that we check without throwing an error.